graph/rewards_test.go

```diff
@@ -1334,3 +1334,111 @@ func (r *RewardsQueryTestSuite) Test_Query_GetUserRewards_BackPaginate_LastBefor
 	}
 	`, beneficiary.Hex(), beneficiary.Hex()), string(b))
 }
+
+func (r *RewardsQueryTestSuite) Test_Query_GetUserRewards_NullEarnings() {
+	currTime := time.Now().UTC().Truncate(time.Second)
+	_, beneficiary, err := test.GenerateWallet()
+	r.NoError(err)
+
+	r.createDependencies()
+
+	// Aftermarket Earnings
+	adEarn, ok := new(big.Int).SetString("1", 10)
+	r.True(ok)
+
+	// Streak Earnings
+	strkEarn, ok := new(big.Int).SetString("1", 10)
+	r.True(ok)
+
+	var rewards = []models.Reward{
+		{
```

Comment:
Might we explicitly write out the zero values here that would have tripped up the old code? It's a bit difficult to notice the omission, there are lots of fields.